src/test/java/duke/mocks/TaskMock.java

```diff
@@ -0,0 +1,21 @@
+package duke.mocks;
+
+import duke.Task;
+public class TaskMock extends Task {
```

Comment:
I really like the usage of stubs when performing your unit testing. Really shows you went the extra mile!